### PR TITLE
Don't build on push

### PR DESCRIPTION
**Pull request by SashaWeinstein**
Taking away this option after an accidental build led to an erroneous qaqc page. Think building on push is too easy to do accidentally. 

Curious what other engineers think but ultimately waiting for Amanda to ok before merging.

### CODE REVIEW

**Review by mbh329 — APPROVED** [review]
I think this is a necessary improvement as we learned when I made an insignificant change to issue template and then had a new version outputted in DO but waiting for @AmandaDoyle to sign off on this before we should merge

**Review by AmandaDoyle — APPROVED** [review]

**Comment by AmandaDoyle** [comment]
Approved.  For where FacDB is as a product we do not need continuous feedback, so builds should be intentional.

**Pull request merged**
